### PR TITLE
[TESTS] Added tests for multiaccount

### LINKED ISSUE #8747: Add tests for adding address to multiaccount instance
# Problem

New feature arrived: you now can add another ethereum address to your multiaccount https://github.com/status-im/status-react/pull/8731

Design: https://www.figma.com/file/XUehMnhyD1FGcWzvGz6SXqvh/Mobile-wallet?node-id=2373%3A1717

## Tests

- [ ] https://ethstatus.testrail.net/index.php?/cases/view/6224 critical

- [ ] https://ethstatus.testrail.net/index.php?/cases/view/6225 high

## Notes

history is not implemented for additional accounts

### PR DESCRIPTION
**Pull request by yevh-berdnyk**
fixes #8747 

**Added tests:**
test_add_account_to_multiaccount_instance
test_send_funds_between_accounts_in_multiaccount_instance

**Fixed test:**
test_fetch_more_history_in_empty_chat

status: ready

### CODE REVIEW

**Review by asemiankevich — APPROVED** [review]

File: test/appium/tests/atomic/account_management/test_wallet_management.py [hunk 1]
```diff
@@ -256,3 +256,33 @@ def test_add_custom_token(self):
...
+        if wallet_view.element_by_text('Account added').is_element_displayed():
+            self.driver.fail('Account is added without password')
+        wallet_view.enter_your_password_input.send_keys('000000')
+        wallet_view.generate_account_button.click()
+        if not wallet_view.element_by_text_part('Password seems to be incorrect').is_element_displayed():
+            self.driver.fail("Incorrect password validation is not performed")
+        wallet_view.enter_your_password_input.clear()
+        wallet_view.enter_your_password_input.send_keys(common_password)
+        wallet_view.generate_account_button.click()
+        account_name = 'test name'
```
Comment (churik) [inline test/appium/tests/atomic/account_management/test_wallet_management.py]: Probably there will be better to generate some random name?

File: test/appium/tests/atomic/transactions/test_wallet.py [hunk 2]
```diff
@@ -410,16 +410,73 @@ def test_can_see_all_transactions_in_history(self):
...
+    @marks.testrail_id(5429)
+    @marks.medium
+    def test_set_currency(self):
+        sign_in_view = SignInView(self.driver)
+        user_currency = 'Euro (EUR)'
+        sign_in_view.create_user()
+        wallet_view = sign_in_view.wallet_button.click()
+        wallet_view.set_currency(user_currency)
+        if not wallet_view.find_text_part('EUR'):
+            pytest.fail('EUR currency is not displayed')
```
Comment (churik) [inline test/appium/tests/atomic/transactions/test_wallet.py]: Not sure why we use here pytest.fail but not self.driver.fail

File: test/appium/tests/atomic/transactions/test_wallet.py [hunk 3]
```diff
@@ -410,16 +410,73 @@ def test_can_see_all_transactions_in_history(self):
...
+        send_transaction.amount_edit_box.set_value(str(transaction_amount_1))
+        send_transaction.confirm()
+        send_transaction.chose_recipient_button.click()
+        send_transaction.accounts_button.click()
+        send_transaction.element_by_text('Status account').click()
+        send_transaction.sign_transaction_button.click()
+        total_fee = send_transaction.get_transaction_fee_total()
+        send_transaction.sign_transaction()
+        if not wallet_view.element_by_text('Transaction sent').is_element_displayed():
+            pytest.fail('Transaction was not sent from the new account')
```
Comment (churik) [inline test/appium/tests/atomic/transactions/test_wallet.py]: The same here - what is the best option  pytest.fail or self.driver.fail?

File: test/appium/tests/atomic/transactions/test_wallet.py [hunk 4]
```diff
@@ -410,16 +410,73 @@ def test_can_see_all_transactions_in_history(self):
...
+        updated_balance = self.network_api.get_balance(address)
+
+        wallet_view.send_transaction_button.click()
+        send_transaction.amount_edit_box.click()
+        transaction_amount_1 = float(transaction_amount) * 0.7
+        send_transaction.amount_edit_box.set_value(str(transaction_amount_1))
+        send_transaction.confirm()
+        send_transaction.chose_recipient_button.click()
+        send_transaction.accounts_button.click()
+        send_transaction.element_by_text('Status account').click()
```
Comment (churik) [inline test/appium/tests/atomic/transactions/test_wallet.py:467]: 'Status account' is main account, worth to add it to wallet_view?

Comment (yevh-berdnyk) [reply test/appium/tests/atomic/transactions/test_wallet.py:467]: I think no because in most cases we send funds from it unlike in that case

**Review by churik — APPROVED** [review]

**Pull request merged**